### PR TITLE
Associations

### PR DESCRIPTION
**Pull request by Cerdtruir**

### CODE REVIEW

**Review by ikraamg — CHANGES_REQUESTED** [review]
Looking good 👍 

I am so used to using Rails that I did not even consider how to create associations in plain Ruby.

A couple of things still need to be added to complete the associations:
Book Class:
- A method for adding rentals should be present.

Person Class
- A method for adding rentals should be present.

**Comment by Cerdtruir** [comment]
> Looking good +1
> 
> I am so used to using Rails that I did not even consider how to create associations in plain Ruby.
> 
> A couple of things still need to be added to complete the associations: Book Class:
> 
> * A method for adding rentals should be present.
> 
> Person Class
> 
> * A method for adding rentals should be present.

Is it fine now?

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def rental(person, date)
```
Comment (ikraamg) [inline book.rb]:
```suggestion
  def add_rental(person, date)
```

File: person.rb [hunk 2]
```diff
@@ -22,6 +23,10 @@ def validate_name
     @name = @corrector.correct_name(@name)
   end
 
+  def rental(book, date)
```
Comment (ikraamg) [inline person.rb]:
```suggestion
  def add_rental(book, date)
```

**Review by ikraamg — APPROVED** [review]
Yeah looks good now 👍 

The method naming suggestion suits the class pattern created in Classroom class.

**Pull request merged**